.github/workflows/build-ci.yml

```diff
@@ -30,6 +28,7 @@ jobs:
 
         steps:
             -   uses: actions/checkout@v4
+
```

Comment:
Could you revert this empty newlines, it forces you to scroll more than necessary. The `-` already make each task legible.